package.json

```diff
@@ -145,7 +145,7 @@
     "jaeger-client": "^3.0.0",
     "kafka-node": "^5.0.0",
     "log4js": "^6.0.0",
-    "mqtt": "^4.0.0",
```

Comment:
Please keep the previous version as well here, because the implementation supports both version.
`"^4.0.0 || ^5.0.0"`